### PR TITLE
Feature/saf fixes

### PR DESCRIPTION
**Pull request by Safihre**
More fixes..

### CODE REVIEW

File: sabnzbd/newsunpack.py [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ def PAR_Verify(parfile, parfile_nzf, nzo, setname, joinables, classic=False, sin
         # Allow options if not classic or when classic and non-classic are the same
         if options and (not classic or (PAR2_COMMAND == PAR2C_COMMAND)):
             command.insert(2, options)
-    logging.debug('Par2-classic = %s', classic)
+    logging.debug('Par2-classic/cmdline = %s', classic)
```
Comment (shypike) [inline sabnzbd/newsunpack.py]: Not just "par2cmdline"?

File: sabnzbd/newsunpack.py [hunk 2]
```diff
@@ -1572,7 +1572,7 @@ def PAR_Verify(parfile, parfile_nzf, nzo, setname, joinables, classic=False, sin
         used_joinables.extend(reconstructed)
 
     if retry_classic:
-        logging.debug('Retry PAR2-joining with par2-classic')
+        logging.debug('Retry PAR2-joining with par2-classic/cmdline')
```
Comment (shypike) [inline sabnzbd/newsunpack.py]: Same here

File: win/par2/README_par2cmdline.txt [hunk 3]
```diff
@@ -0,0 +1,336 @@
+par2cmdline is a PAR 2.0 compatible file verification and repair tool.
```
Comment (shypike) [inline win/par2/README_par2cmdline.txt]: This file should be called "Licence_par2cmdline.txt" and the other one (for multi-core) "Licence_par2.txt". "COPYING" should be renamed to "GPL2.txt".
The x64 folder has the same issue.

File: README.md [hunk 4]
```diff
@@ -24,14 +24,17 @@ SABnzbd has a good deal of dependencies you'll need before you can get running.
 
 - `python` (only 2.7.x and higher, but not 3.x.x)
 - `python-cheetah`
-- `python-dbus`
```
Comment (shypike) [inline README.md]: Are these no longer a requirement??

File: INSTALL.txt [hunk 5]
```diff
@@ -68,19 +68,21 @@ Windows
 Essential modules
     cheetah-2.0.1+          use "pip install cheetah"
     par2cmdline >= 0.4      http://parchive.sourceforge.net/
+                            Beware effect of -N parameter introduced in v0.6.13
```
Comment (shypike) [inline INSTALL.txt]: "Beware". How is the user to interpret this? Do we have a link to more info?

File: sabnzbd/newsunpack.py (hunk 1 above)
Comment (Safihre) [inline sabnzbd/newsunpack.py]: No because on Mac it is still par2-classic..

File: README.md (hunk 4 above)
Comment (Safihre) [inline README.md]: python-dbus, cryptography, unzip, 7zip and yenc are not required, SAB will work without them.
Just missing some features.
Now it's the same as in INSTALL.txt:
https://github.com/sabnzbd/sabnzbd/blob/master/INSTALL.txt

File: INSTALL.txt (hunk 5 above)
Comment (Safihre) [inline INSTALL.txt]: In the current release (v0.6.13) -N will include missing blocks, but in upcoming release the par2cmdline guy has inverted the behavior and -N will exclude missing blocks. Very confusing..

File: README.md (hunk 4 above)
Comment (shypike) [inline README.md]: OK, makes sense

File: sabnzbd/newsunpack.py (hunk 2 above)
Comment (thezoggy) [inline sabnzbd/newsunpack.py]: would think it should be `par2-classic/par2cmdline`

**Pull request merged**